fastlane/Fastfile

```diff
@@ -101,8 +113,20 @@ end
 
 def prepare_git(branch)
 	ensure_git_status_clean
-	ensure_git_branch(branch:branch)
-	git_pull
+
+	# ensure_git_branch(branch:branch)
+	masterHash = sh("git rev-parse origin/master")
```

Comment:
This will only work on master. But prepare_git was supposed to be used with any branch. Maybe change this to sh("git rev-parse origin/YOUR_BRANCH")